src/styles/blocks/menu.scss

```diff
@@ -0,0 +1,41 @@
+.menu {
+  box-sizing: border-box;
```

Comment:
Consider removing 'box-sizing: border-box;' since it's a better practice to apply it globally, for example, in the main.scss using the following code: *, *::before, *::after { box-sizing: border-box; }